core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -191,7 +207,17 @@ oppia.directive('audioControls', [
           };
 
           $scope.openAudioTranslationSettings = function() {
-            AudioTranslationManagerService.showAudioTranslationSettingsModal();
+            AudioTranslationManagerService.showAudioTranslationSettingsModal(
+              function(newLanguageCode) {
+                if ($scope.isAudioAvailableInCurrentLanguage() &&
+                    !isAutogeneratedLanguageCodeSelected()) {
+                  $scope.audioIsLoading = true;
+                  var audioTranslation = getAudioTranslationInCurrentLanguage();
```

Comment:
If I'm reading correctly, this doesn't seem to get used anywhere?

Reply:
It gets used now